tests/baselines/reference/docker/office-ui-fabric.log

```diff
@@ -10,29 +10,29 @@ Starting "rush rebuild"
 Executing a maximum of 1 simultaneous processes...
 
 [@uifabric/prettier-rules] started
-1 of 40: [@uifabric/prettier-rules] completed successfully in ? seconds
```

Comment:
@weswigham we might want to anonymize these numbers too. We would still see diffs when office-ui-fabric adds build steps, but they would be smaller.

Reply:
Their ordering and count _is_ implied by the output log order (since `rush` parallelism is set to 1)